 <launch>
   <node name="cortex_bridge" type="cortex_bridge" pkg="cortex_bridge" output="screen">
     <param name="local_ip" value="10.1.1.200"/>
     <param name="cortex_ip" value="10.1.1.190"/>
   </node>
 </launch>
